src/faucet.js

```diff
@@ -1,45 +1,49 @@
 const Web3 = require("web3");
 require('dotenv').config();
 const axios = require("axios");
-const jsonInterface = require('../src/abi.json');
-const { SSV_INFURA_HTTPS_ENDPOINT, SSV_EXPLORER_URL, SINGER_PRIVATE_KEY, SIGNER_OWNER_ADDRESS, SSV_CONTRACT_ADDRESS } = process.env;
-const web3 = new Web3(new Web3.providers.HttpProvider(SSV_INFURA_HTTPS_ENDPOINT));
+const { getContractToken, NETWORK_ABI } = require("./envHelper");
+const {  SSV_EXPLORER_URL, SINGER_PRIVATE_KEY, SIGNER_OWNER_ADDRESS } = process.env;
 
 const faucetApiUrl = `${SSV_EXPLORER_URL}/api/ssv_faucet/`;
 const faucetConfigApiUrl = `${SSV_EXPLORER_URL}/api/ssv_faucet_config/`;
-const contract = new web3.eth.Contract(jsonInterface, SSV_CONTRACT_ADDRESS);
+
+const createWeb3 = (infura) => new Web3(new Web3.providers.HttpProvider(infura));
 
 const getTransactions = async () => {
     console.log('[FAUCET][INFO] Start fetching initiated transactions from Explorer Center')
     try {
-        const faucetConfig = await getFaucetConfig();
-        console.log(`[FAUCET][INFO] faucet config: ${JSON.stringify(faucetConfig)}`);
-        const faucetBalance = await getBalance(contract, SIGNER_OWNER_ADDRESS);
-        console.log(`[FAUCET][INFO] faucet balance: ${faucetBalance}`);
         let response = (await axios.get(faucetApiUrl + '?status=initiated')).data;
-        const transactionsCapacity = Math.floor(faucetBalance / faucetConfig?.amount_to_transfer) - +response?.length;
-        console.log(`[FAUCET][INFO] transactions capacity: ${transactionsCapacity}`);
-        if(faucetConfig.transactions_capacity !== transactionsCapacity) {
-            console.log(`[FAUCET][INFO] update faucet config: ${transactionsCapacity}`);
-            await updateFaucetConfig(faucetConfig, transactionsCapacity)
-        }
         if(+response?.length === 0) console.log(`[FAUCET][INFO] No transaction to execute... starting over in 2 seconds`)
         for (let index = 0; index < response.length; index++) {
+            const { network } = response[index];
+            const networkData = getContractToken(network);
+            const web3 = createWeb3(networkData.infura);
+            const contract = new web3.eth.Contract(NETWORK_ABI[network], networkData.tokenAddress);
+            const faucetConfig = await getFaucetConfig();
```

Comment:
Faucet config should be per network also, it can have different balances in prater or holesky